test/uk/gov/hmrc/traderservices/controllers/ExportQuestionsFormSpec copy.scala

```diff
@@ -0,0 +1,91 @@
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.traderservices.controllers
+
+import play.api.data.FormError
+import uk.gov.hmrc.play.test.UnitSpec
+import uk.gov.hmrc.traderservices.models.{ExportFreightType, ExportPriorityGoods, ExportRequestType, ExportRouteType}
+import uk.gov.hmrc.traderservices.support.FormMatchers
+import play.api.data.Form
+
+class ExportQuestionsFormSpec extends UnitSpec with FormMatchers {
+
+  def validate[A](form: Form[A], formInput: Map[String, String], formOutput: A): Unit = {
+    form.bind(formInput).value shouldBe Some(formOutput)
+    form.fill(formOutput).data shouldBe formInput
+  }
+
+  def validate[A](form: Form[A], fieldName: String, formInput: Map[String, String], errors: Seq[String]): Unit = {
+    form.bind(formInput).value shouldBe None
+    form.bind(formInput).errors should haveOnlyErrors(errors.map(e => FormError(fieldName, e)): _*)
+  }
+
+  "Export questions forms" should {
+
+    "bind some requestType and return ExportRequestType and fill it back" in {
+      val form = TraderServicesFrontendController.ExportRequestTypeForm
+      validate(form, Map("requestType" -> "New"), ExportRequestType.New)
+      validate(form, Map("requestType" -> "Hold"), ExportRequestType.Hold)
+      validate(form, Map("requestType" -> "Cancellation"), ExportRequestType.Cancellation)
+      validate(form, Map("requestType" -> "C1601"), ExportRequestType.C1601)
+      validate(form, Map("requestType" -> "C1602"), ExportRequestType.C1602)
+      validate(form, Map("requestType" -> "C1603"), ExportRequestType.C1603)
+      validate(form, Map("requestType" -> "WithdrawalOrReturn"), ExportRequestType.WithdrawalOrReturn)
+      validate(form, "requestType", Map(), Seq("error.exportRequestType.required"))
+      validate(form, "requestType", Map("requestType" -> "Foo"), Seq("error.exportRequestType.invalid-option"))
+    }
+
+    "bind some routeType and return ExportRouteType and fill it back" in {
+      val form = TraderServicesFrontendController.ExportRouteTypeForm
+      validate(form, Map("routeType" -> "Route1"), ExportRouteType.Route1)
+      validate(form, Map("routeType" -> "Route1Cap"), ExportRouteType.Route1Cap)
+      validate(form, Map("routeType" -> "Route2"), ExportRouteType.Route2)
+      validate(form, Map("routeType" -> "Route3"), ExportRouteType.Route3)
+      validate(form, Map("routeType" -> "Route6"), ExportRouteType.Route6)
+      validate(form, Map("routeType" -> "Hold"), ExportRouteType.Hold)
+      validate(form, "routeType", Map(), Seq("error.exportRouteType.required"))
+      validate(form, "routeType", Map("routeType" -> "Foo"), Seq("error.exportRouteType.invalid-option"))
+    }
+
+    "bind tes/no and return Boolean and fill it back" in {
+      val form = TraderServicesFrontendController.ExportHasPriorityGoodsForm
+      validate(form, Map("hasPriorityGoods" -> "yes"), true)
+      validate(form, Map("hasPriorityGoods" -> "no"), false)
+    }
```

Comment:
I noticed in the formFieldMappings spec you have tested to get the error for no option entered so it might not add any value here but is is worth it (or even possible) to have lines on this like the others do? ie to test left blank or invalid option? This is a genuine question so completely fine if you say no or it's not worth it